### PR TITLE
[SKIP-PREVIEW]- ROC-3859: Expenses-other

### PR DESCRIPTION
**Pull request by varma02**
### Change description

expenses other

### CODE REVIEW

File: src/main/public/js/append-income-source-section.js [hunk 1]
```diff
@@ -32,3 +34,9 @@ $(document).ready(function () {
 
   feature.init();
 });
+
+
+// var lastRow = $('.panel panel-border-narrow expandable').last()
```
Comment (timja-kainos) [inline src/main/public/js/append-income-source-section.js]: remove commented out js

File: src/main/public/js/append-income-source-section.js [hunk 2]
```diff
@@ -17,12 +17,14 @@ $(document).ready(function () {
...
 
     var setup = function () {
-      addOtherIncomeSourceButtonElement.keyup(addNewOtherIncome);
-      otherIncomeExpenseSourceSectionElement.change();
+      addOtherIncomeSourceButtonElement.click(addNewOtherIncome);
     }
 
     var addNewOtherIncome = function () {
-
+      console.log('this is a test')
```
Comment (timja-kainos) [inline src/main/public/js/append-income-source-section.js]: remove console.log

File: src/main/features/response/routes/statement-of-means/monthly-expenses.ts [hunk 3]
```diff
@@ -58,14 +58,14 @@ function actionHandler (req: express.Request, res: express.Response, next: expre
     const form: Form<MonthlyExpenses> = req.body
 
     switch (actionName) {
-      case 'addOther':
-        form.model.addEmptyOtherIncome()
+      case 'addOtherExpenseSource':
```
Comment (timja-kainos) [inline src/main/features/response/routes/statement-of-means/monthly-expenses.ts]: is the source bit needed? nothing in expenses code has source in it

**Comment by varma02** [comment]
duplicate

**Pull request closed without merging**